scripting/get5_apistats.sp

```diff
@@ -35,14 +35,20 @@
 #pragma newdecls required
 
 int g_MatchID = -1;
-
+ConVar g_UseSVGCvar;
 ConVar g_APIKeyCvar;
 char g_APIKey[128];
 
 ConVar g_APIURLCvar;
 char g_APIURL[128];
 
-#define LOGO_DIR "resource/flash/econ/tournaments/teams"
+//This logo dir its for old csgo hud
+    if (g_UseSVGCvar.BoolValue) {
+      #define LOGO_DIR "materials/panorama/images/tournaments/teams"
+    } else {
```

Comment:
This doesn't work  (or compile), can you do something like:

```
#define LEGACY_LOGO_DIR  "resource/flash/econ/tournaments/teams"
#define LOGO_DIR  "materials/panorama/images/tournaments/teams"
```

Then you can do

```
  if (g_UseSVGCvar.BoolValue) {
    Format(logoPath, sizeof(logoPath), "%s/%s.svg", LOGO_DIR, logo);
  } else {
    Format(logoPath, sizeof(logoPath), "%s/%s.png", LEGACY_LOGO_DIR , logo);
  }
```

Reply:
i saw it on same day but it's too late for me